src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -104,6 +104,11 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice web config update', 'java_container_version', help="The version of the java container, e.g., '8.0.23' for Tomcat")
 register_cli_argument('appservice web config update', 'app_command_line', options_list=('--startup-file',), help="The startup file for linux hosted web apps, e.g. 'process.json' for Node.js web")
 
+register_cli_argument('appservice web config ssl bind', 'ssl_type', help='The ssl cert type, e.g., SNI or IP', **enum_choice_list(['SNI', 'IP']))
```

Comment:
Don't need `e.g., SNI or IP` in help string as you define a choice list anyway.
As is, help would show:
```
--ssl-type : The ssl cert type, e.g., SNI or IP. Allowed values: SNI, IP.
```